easybuild/easyblocks/i/impi.py

```diff
@@ -58,6 +58,12 @@ def extra_options():
         }
         return IntelBase.extra_options(extra_vars)
 
+    def prepare_step(self):
+        if LooseVersion(self.version) >= LooseVersion('2017.2.174'):
+            super(EB_impi, self).prepare_step(requires_runtime_license=False)
```

Comment:
Eek, abort, abort! That is a named user licence, not valid for HPC:

> iii. NAMED-USER LICENSE: If You obtained the Materials under a named-user license, You may
allow only one (1) individual to install and use the Materials on no more than three (3)
computers provided that same individual is using the Materials only on one (1) computer at a
time. If You obtained a time-limited named-user license, the term of Your license and your
ability to use the Materials is limited to the time period of the obtained license, which is specified
on Intel’s download website, specified in the applicable documentation or controlled by the
license key for the Materials.